package.json

```diff
@@ -38,11 +38,22 @@
     "node": "^8.10.0 || ^10.13.0 || >=11.10.1"
   },
   "dependencies": {
-    "codependency": "^2.1.0",
     "json-stringify-safe": "^5.0.1",
     "lodash": "^4.17.15",
     "reconnect-core": "^1.3.0"
   },
+  "peerDependencies": {
+    "winston": "^3.2.1",
```

Comment:
Just curious here but what does the version pin do, in this case? Since it's a peer dependency and as such is not installed by this package, right?
Will it throw up an error now if a customer is using 3.1.x?

Reply:
The NPM docs actually seem to indicate that this is the case;
> Assuming the host complies with [semver](https://semver.org/), only changes in the host package's major version will break your plugin. Thus, if you've worked with every 1.x version of the host package, use "^1.0" or "1.x" to express this. If you depend on features introduced in 1.5.2, use "^1.5.2".
> https://docs.npmjs.com/cli/v7/configuring-npm/package-json#peerdependencies

I think we may need to figure out if we need a specific version or if just 3.x will do 😅

Reply:
I've just checked with version 3.0.0 at least and it works